ng/src/web/pages/scanconfigs/editconfigfamilydialog.js

```diff
@@ -95,7 +96,7 @@ class Nvt extends React.Component {
               _('default') :
               timeout
           }
-          {!(is_empty(default_timeout)) ?
+          {!is_empty(default_timeout) ? // eslint-disable-line no-negated-condition
               ' (' + default_timeout + ')' :
               ''
           }
```

Comment:
Could be changed to
```jsx
{is_empty(default_timeout) ?
  '' : ' (' + default_timeout + ')'
}
```
to avoid the eslint directive

Reply:
Done